packages/core/src/components/select/select.scss

```diff
@@ -160,6 +160,10 @@
     }
   }
 
+  ix-dropdown {
+    max-width: 100%;
```

Comment:
`Defines the max-width as a percentage of the containing block's width` this means the ix-select host will be the 100%. 
Or did I see this wrong?

Reply:
Correct its not the case here